exercises/02-storage.md

```diff
@@ -37,3 +38,17 @@ Create a server using the API and add a volume to it. Using the Cloud Control Pa
 Can you grow the size of the volume without restarting the system?
 
 Hint: Click on the question mark in the Cloud Control Panel near each volume for further instructions.
+
+---
+
+### `02.03` - Snapshot a Volume
+
+> Please be aware: Snapshots do not serve as a substitute for backups.
+
+Using the Cloud Control Panel, or your favourite tool, create a server with a root volume attached.
```

Comment:
> create a server with a root volume attached.

Since you cannot start a server without root volume, I would just write: Create a server.